make/otp_subdir.mk

```diff
@@ -56,3 +56,6 @@ opt debug lcnt release docs release_docs tests release_tests clean depend valgri
 	    fi	;							\
 	    echo "=== Leaving application" `basename $$app_pwd` ;	\
 	fi
+
+test:
```

Comment:
Does this really have to be in subdir.mk? Feels very out of place.

Reply:
It was the best place I could find. Placing it in `make/otp.mk.in` or in `make/target.mk` makes "make test" the default target (e.g., in the root).

Reply:
Maybe you can define `.DEFAULT_GOAL` to ensure the correct target is the default. Not sure it works across all Makes.

Reply:
I have added a file called `make/app_target.mk` that contains the test target. This file is included in the relevant make files (see commit below).

Reply:
@essen The `.DEFAULT_GOAL` idea could probably also work but I think it would require that `.DEFAULT_GOAL`s are added to many make files as the name of the default goal may not always be the same.